src/components/Airplane.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+// import './rh-logo.scss';
```

Comment:
I approved... but what about this SVG inside of the JS... what is up with that?

Reply:
SVG is just same as any other HTML so React can render it without any problems. And because PF is not providing paper plane icon https://github.com/patternfly/patternfly-react/pull/1643 we have to add it here.

Reply:
I mean I get that any XML can be embedded like this, but that might come with a penalty.

I think this embeds the XML inside of the bundle instead of serving the XML as a separate file. Right?

If that is the case we should avoid this, we have different caching rules for different content types and images, pdfs, svg etc should be cached longer than the js bundles.

I think the best argument I've seen to inline them is so that we can use CSS to style them. i.e. we can select a path and change it's color to blue.

Is that the reason we are doing this?

Reply:
Yep you are correct this is part of JS bundle instead of SVG bundle. It's really unfortunate that we didn't had this SVG file before, we could have add it much sooner to PF icons.

I think that the best sollution here would be not to include this svg at all and once that PF request is merged we will revisit all icons in landing page and use icons from react-icons package instead of using svgs.

Reply:
I'm fine with removing this for now and then adding it back in a later PR updated with all PF icons